File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-06/plagiarized/L2/07/Kasus6L2.kt
package IR

import java.util.Scanner

fun main(args: Array<String>) {
    // Deklarasi
    val inp = Scanner(System.`in`)
    val arrNum = IntArray(10)
    // for untuk 10 kali input
    for (i in 0 until 10) {
        // Read a number
        print("Read a number: ")
        arrNum[i] = inp.nextInt()
    }
    // Display the array
    for (i in 9 downTo 0) {
        // Print angka dalam array
        println(arrNum[i])
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L2/07/Kasus6L2.py
import sys

# Deklarasi
arr_num = [0] * 10

# for untuk 10 kali input
for i in range(10):
    # Read a number
    print("Read a number: ", end="")
    arr_num[i] = int(input())

# Display the array
for i in range(9, -1, -1):
    # Print angka dalam array
    print(arr_num[i])

